projects/netgrif-components/src/lib/side-menu/content-components/user-assign/user-assign.component.html

```
@@ -2,11 +2,11 @@
     <mat-toolbar color="primary">{{ 'side-menu.user.assign' | translate | uppercase }}</mat-toolbar>
     <div class="user-assign-container side-menu-root" fxLayout="column" fxFlex>
         <div fxLayout="row" fxLayoutAlign="start center" class="netgrif-input netgrif-input-fix">
-            <mat-icon fxFlex class="search-icon">search</mat-icon>
             <mat-form-field fxFlex="100" class="user-assign-search" color="primary" appearance="outline">
+                <mat-icon matPrefix class="prefix-icon">search</mat-icon>
                 <input matInput placeholder="{{ 'side-menu.user.choose' | translate }}"
                        [formControl]="searchUserControl">
-                <button mat-button *ngIf="searchUserControl.value" matSuffix mat-icon-button
+                <button *ngIf="searchUserControl.value" matSuffix mat-icon-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)